ansible.cfg

```diff
@@ -6,8 +6,16 @@ stdout_callback = yaml
 host_key_checking = False
 ansible_managed = This file is managed by Ansible
   And it was last modified by the user '{uid}'
+remote_user = ec2-user
+private_key_file=/home/ec2-user/.ssh/deploy.pem
+become = true
+become_method: sudo
+become_user: root
+inventory = ansible_plugins
+enable_plugins = aws_ec2 boto3
 
 [ssh_connection]
 #Enable faster networking
 pipelining = True
 ssh_args = -o ControlMaster=auto -o ControlPersist=300s
+scp_if_ssh=True
```

Comment:
Is it really necessary here?

Reply:
Its need To avoid a warnings due ansible tasks